dev-notes/minimal-os-for-fobnail.md

```diff
@@ -0,0 +1,203 @@
+# Minimal OS for Fobnail project
+
+The Fobnail project aims to provide a reference architecture for building
+offline integrity measurement servers on the USB device and clients running in
+Dynamically Launched Measured Environments (DLME). It should allow the Fobnail
+owner to verify the trustworthiness of the running system before performing any
+sensitive operation.
+
+According to the information provided at TrenchBoot Summit 2021 (the fragment
+discussed here - https://youtu.be/xZoCtNV8Qs0?t=5062), the minimum OS, together
+with the connected Fobnail Token, should allow determining the state of the
+platform before the target OS is launched. As we can see in the presentation,
+`heads` would be responsible for the platform's trust control, while in this
+document, we want to present the results of research on the possibilities of
+using various operating systems.
```

Comment:
@TomaszAIR let's link to explanation what heads is. Also, we kind of missing the goal here. Heads is payload/bootloader before running OS, but what we should explore is OS running as DLME. Heads will not be running as DLME, although we may consider that, and it would be useful exercise since Heads is nothing then LinuxBoot with different user space (busybox IIUC). Am I missing something?

Reply:
I linked to theirs repo

Reply:
Yes